scripts/run

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+# run the service in one of different deployment modes
+set -e #-x
+
+usage() {
+  cat <<-'EOF'
+usage: run [-h|--help] RUNTYPE
+[-h|--help]       print this help text
+RUNTYPE is one of:
+- tests           run tests against repo
+- integration     run integration tests against running instance;
+                  either `local` or `docker`
+- local           run as a local terminal process
+- docker          run in a docker container
+- check           run linter and checks
+EOF
+}
+
+# script variables and defaults
+source "$(dirname "$0")"/init.sh
+
+AZURE_ENV=dev
+
+# process command-line options (if any)
+die() { log_error "$*" >&2; echo ""; usage; exit 2; }  # complain to STDERR and exit with error
+needs_arg() { if [ -z "$OPTARG" ]; then die "No arg for --$OPT option"; fi; }
+
+while getopts h-: OPT; do
+  # support long options: https://stackoverflow.com/a/28466267/519360
+  if [ "$OPT" = "-" ]; then   # long option: reformulate OPT and OPTARG
+    OPT="${OPTARG%%=*}"       # extract long option name
+    OPTARG="${OPTARG#"$OPT"}"   # extract long option argument (may be empty)
+    OPTARG="${OPTARG#=}"      # if long option argument, remove assigning `=`
+  fi
+  case "$OPT" in
+    h | help )     usage; exit 0 ;;
+    \? )           usage; exit 2 ;;  # bad short option (error reported via getopts)
+    * )            die "Illegal option --$OPT" ;;            # bad long option
+  esac
+done
+shift $((OPTIND-1)) # remove parsed options and args from $@ list
+
+
+# process positional arguments (if any)
+RUNTYPE=${1}
+
+run_tests() {
+  eval "$("${SCRIPTS_DIR}"/render-configs.sh -a "$AZURE_ENV" -e)"
```

Comment:
I don't think unit tests need `render_configs.sh` to run. In fact, I don't think they need any env vars set at all.